### DISCUSSION TITLE
disable rekeying

### DISCUSSION THREAD

**Discussion by musaleh17** [strongswan/strongswan#1630] [Q&A] [answered]
What is the best way for disabling rekey?

**Comment by tobiasbrunner** [comment, marked as answer]
Set `rekey_time = 0`, see [the docs](https://docs.strongswan.org/docs/5.9/config/rekeying.html) for details.

But why disable rekeying?

**Reply by musaleh17** [reply]
I don't want to rekey automatically, only if it was triggered by swanctl rekey command.

**Reply by tobiasbrunner** [reply]
Why?
